package.json

```diff
@@ -15,12 +15,6 @@
       "examples/jest/*"
     ]
   },
-  "resolutions": {
-    "@cypress/schematic/@angular-devkit/architect": ">=0.2000.0 < 0.2100.0",
-    "@cypress/schematic/@angular-devkit/core": "^20.0.0",
-    "@cypress/schematic/@angular-devkit/schematics": "^20.0.0",
-    "@cypress/schematic/@schematics/angular": "^20.0.0"
-  },
```

Comment:
Couldn't get the reason for these resolutions - looks like everything works fine without them.